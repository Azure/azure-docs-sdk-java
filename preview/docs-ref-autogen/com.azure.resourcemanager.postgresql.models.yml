### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.postgresql.models"
fullName: "com.azure.resourcemanager.postgresql.models"
name: "com.azure.resourcemanager.postgresql.models"
summary: "Package containing the data models for PostgreSqlManagementClient. The Microsoft Azure management API provides\n create, read, update, and delete functionality for Azure PostgreSQL resources including servers, databases, firewall\n rules, VNET rules, security alert policies, log files and configurations with new business model."
classes:
- "com.azure.resourcemanager.postgresql.models.CreateMode"
- "com.azure.resourcemanager.postgresql.models.DatabaseListResult"
- "com.azure.resourcemanager.postgresql.models.ErrorAdditionalInfo"
- "com.azure.resourcemanager.postgresql.models.FirewallRuleListResult"
- "com.azure.resourcemanager.postgresql.models.GeoRedundantBackup"
- "com.azure.resourcemanager.postgresql.models.IdentityType"
- "com.azure.resourcemanager.postgresql.models.InfrastructureEncryption"
- "com.azure.resourcemanager.postgresql.models.LogFileListResult"
- "com.azure.resourcemanager.postgresql.models.MinimalTlsVersionEnum"
- "com.azure.resourcemanager.postgresql.models.NameAvailabilityRequest"
- "com.azure.resourcemanager.postgresql.models.Operation"
- "com.azure.resourcemanager.postgresql.models.OperationDisplay"
- "com.azure.resourcemanager.postgresql.models.OperationOrigin"
- "com.azure.resourcemanager.postgresql.models.PerformanceTierListResult"
- "com.azure.resourcemanager.postgresql.models.PerformanceTierServiceLevelObjectives"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnectionListResult"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointProperty"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointProvisioningState"
- "com.azure.resourcemanager.postgresql.models.PrivateLinkResourceListResult"
- "com.azure.resourcemanager.postgresql.models.PrivateLinkResourceProperties"
- "com.azure.resourcemanager.postgresql.models.PrivateLinkServiceConnectionStateActionsRequire"
- "com.azure.resourcemanager.postgresql.models.PrivateLinkServiceConnectionStateProperty"
- "com.azure.resourcemanager.postgresql.models.PrivateLinkServiceConnectionStateStatus"
- "com.azure.resourcemanager.postgresql.models.PublicNetworkAccessEnum"
- "com.azure.resourcemanager.postgresql.models.ResourceIdentity"
- "com.azure.resourcemanager.postgresql.models.SecurityAlertPolicyName"
- "com.azure.resourcemanager.postgresql.models.ServerAdministratorResourceListResult"
- "com.azure.resourcemanager.postgresql.models.ServerForCreate"
- "com.azure.resourcemanager.postgresql.models.ServerKeyListResult"
- "com.azure.resourcemanager.postgresql.models.ServerKeyType"
- "com.azure.resourcemanager.postgresql.models.ServerListResult"
- "com.azure.resourcemanager.postgresql.models.ServerPrivateEndpointConnection"
- "com.azure.resourcemanager.postgresql.models.ServerPrivateEndpointConnectionProperties"
- "com.azure.resourcemanager.postgresql.models.ServerPrivateLinkServiceConnectionStateProperty"
- "com.azure.resourcemanager.postgresql.models.ServerPropertiesForCreate"
- "com.azure.resourcemanager.postgresql.models.ServerPropertiesForDefaultCreate"
- "com.azure.resourcemanager.postgresql.models.ServerPropertiesForGeoRestore"
- "com.azure.resourcemanager.postgresql.models.ServerPropertiesForReplica"
- "com.azure.resourcemanager.postgresql.models.ServerPropertiesForRestore"
- "com.azure.resourcemanager.postgresql.models.ServerState"
- "com.azure.resourcemanager.postgresql.models.ServerUpdateParameters"
- "com.azure.resourcemanager.postgresql.models.ServerVersion"
- "com.azure.resourcemanager.postgresql.models.Sku"
- "com.azure.resourcemanager.postgresql.models.SkuTier"
- "com.azure.resourcemanager.postgresql.models.StorageAutogrow"
- "com.azure.resourcemanager.postgresql.models.StorageProfile"
- "com.azure.resourcemanager.postgresql.models.TagsObject"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRuleListResult"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRuleState"
enums:
- "com.azure.resourcemanager.postgresql.models.AdministratorType"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicyState"
- "com.azure.resourcemanager.postgresql.models.SslEnforcementEnum"
interfaces:
- "com.azure.resourcemanager.postgresql.models.CheckNameAvailabilities"
- "com.azure.resourcemanager.postgresql.models.Configuration"
- "com.azure.resourcemanager.postgresql.models.Configuration.Definition"
- "com.azure.resourcemanager.postgresql.models.Configuration.DefinitionStages"
- "com.azure.resourcemanager.postgresql.models.Configuration.DefinitionStages.Blank"
- "com.azure.resourcemanager.postgresql.models.Configuration.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.postgresql.models.Configuration.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.postgresql.models.Configuration.DefinitionStages.WithSource"
- "com.azure.resourcemanager.postgresql.models.Configuration.DefinitionStages.WithValue"
- "com.azure.resourcemanager.postgresql.models.Configuration.Update"
- "com.azure.resourcemanager.postgresql.models.Configuration.UpdateStages"
- "com.azure.resourcemanager.postgresql.models.Configuration.UpdateStages.WithSource"
- "com.azure.resourcemanager.postgresql.models.Configuration.UpdateStages.WithValue"
- "com.azure.resourcemanager.postgresql.models.ConfigurationListResult"
- "com.azure.resourcemanager.postgresql.models.Configurations"
- "com.azure.resourcemanager.postgresql.models.Database"
- "com.azure.resourcemanager.postgresql.models.Database.Definition"
- "com.azure.resourcemanager.postgresql.models.Database.DefinitionStages"
- "com.azure.resourcemanager.postgresql.models.Database.DefinitionStages.Blank"
- "com.azure.resourcemanager.postgresql.models.Database.DefinitionStages.WithCharset"
- "com.azure.resourcemanager.postgresql.models.Database.DefinitionStages.WithCollation"
- "com.azure.resourcemanager.postgresql.models.Database.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.postgresql.models.Database.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.postgresql.models.Database.Update"
- "com.azure.resourcemanager.postgresql.models.Database.UpdateStages"
- "com.azure.resourcemanager.postgresql.models.Database.UpdateStages.WithCharset"
- "com.azure.resourcemanager.postgresql.models.Database.UpdateStages.WithCollation"
- "com.azure.resourcemanager.postgresql.models.Databases"
- "com.azure.resourcemanager.postgresql.models.FirewallRule"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.Definition"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.DefinitionStages"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.DefinitionStages.Blank"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.DefinitionStages.WithEndIpAddress"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.DefinitionStages.WithStartIpAddress"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.Update"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.UpdateStages"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.UpdateStages.WithEndIpAddress"
- "com.azure.resourcemanager.postgresql.models.FirewallRule.UpdateStages.WithStartIpAddress"
- "com.azure.resourcemanager.postgresql.models.FirewallRules"
- "com.azure.resourcemanager.postgresql.models.LocationBasedPerformanceTiers"
- "com.azure.resourcemanager.postgresql.models.LogFile"
- "com.azure.resourcemanager.postgresql.models.LogFiles"
- "com.azure.resourcemanager.postgresql.models.NameAvailability"
- "com.azure.resourcemanager.postgresql.models.OperationListResult"
- "com.azure.resourcemanager.postgresql.models.Operations"
- "com.azure.resourcemanager.postgresql.models.PerformanceTierProperties"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.Definition"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.DefinitionStages"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.DefinitionStages.WithPrivateEndpoint"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.DefinitionStages.WithPrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.Update"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.UpdateStages"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnection.UpdateStages.WithTags"
- "com.azure.resourcemanager.postgresql.models.PrivateEndpointConnections"
- "com.azure.resourcemanager.postgresql.models.PrivateLinkResource"
- "com.azure.resourcemanager.postgresql.models.PrivateLinkResources"
- "com.azure.resourcemanager.postgresql.models.RecoverableServerResource"
- "com.azure.resourcemanager.postgresql.models.RecoverableServers"
- "com.azure.resourcemanager.postgresql.models.Replicas"
- "com.azure.resourcemanager.postgresql.models.Server"
- "com.azure.resourcemanager.postgresql.models.Server.Definition"
- "com.azure.resourcemanager.postgresql.models.Server.DefinitionStages"
- "com.azure.resourcemanager.postgresql.models.Server.DefinitionStages.Blank"
- "com.azure.resourcemanager.postgresql.models.Server.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.postgresql.models.Server.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.postgresql.models.Server.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.postgresql.models.Server.DefinitionStages.WithProperties"
- "com.azure.resourcemanager.postgresql.models.Server.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.postgresql.models.Server.DefinitionStages.WithSku"
- "com.azure.resourcemanager.postgresql.models.Server.DefinitionStages.WithTags"
- "com.azure.resourcemanager.postgresql.models.Server.Update"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithAdministratorLoginPassword"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithMinimalTlsVersion"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithPublicNetworkAccess"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithReplicationRole"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithSku"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithSslEnforcement"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithStorageProfile"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithTags"
- "com.azure.resourcemanager.postgresql.models.Server.UpdateStages.WithVersion"
- "com.azure.resourcemanager.postgresql.models.ServerAdministratorResource"
- "com.azure.resourcemanager.postgresql.models.ServerAdministrators"
- "com.azure.resourcemanager.postgresql.models.ServerBasedPerformanceTiers"
- "com.azure.resourcemanager.postgresql.models.ServerKey"
- "com.azure.resourcemanager.postgresql.models.ServerKey.Definition"
- "com.azure.resourcemanager.postgresql.models.ServerKey.DefinitionStages"
- "com.azure.resourcemanager.postgresql.models.ServerKey.DefinitionStages.Blank"
- "com.azure.resourcemanager.postgresql.models.ServerKey.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.postgresql.models.ServerKey.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.postgresql.models.ServerKey.DefinitionStages.WithServerKeyType"
- "com.azure.resourcemanager.postgresql.models.ServerKey.DefinitionStages.WithUri"
- "com.azure.resourcemanager.postgresql.models.ServerKey.Update"
- "com.azure.resourcemanager.postgresql.models.ServerKey.UpdateStages"
- "com.azure.resourcemanager.postgresql.models.ServerKey.UpdateStages.WithServerKeyType"
- "com.azure.resourcemanager.postgresql.models.ServerKey.UpdateStages.WithUri"
- "com.azure.resourcemanager.postgresql.models.ServerKeys"
- "com.azure.resourcemanager.postgresql.models.ServerParameters"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicies"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.Definition"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.Blank"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.WithDisabledAlerts"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.WithEmailAccountAdmins"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.WithEmailAddresses"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.WithRetentionDays"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.WithState"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.WithStorageAccountAccessKey"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.DefinitionStages.WithStorageEndpoint"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.Update"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.UpdateStages"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.UpdateStages.WithDisabledAlerts"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.UpdateStages.WithEmailAccountAdmins"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.UpdateStages.WithEmailAddresses"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.UpdateStages.WithRetentionDays"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.UpdateStages.WithState"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.UpdateStages.WithStorageAccountAccessKey"
- "com.azure.resourcemanager.postgresql.models.ServerSecurityAlertPolicy.UpdateStages.WithStorageEndpoint"
- "com.azure.resourcemanager.postgresql.models.Servers"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.Definition"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.DefinitionStages"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.DefinitionStages.Blank"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.DefinitionStages.WithIgnoreMissingVnetServiceEndpoint"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.DefinitionStages.WithVirtualNetworkSubnetId"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.Update"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.UpdateStages"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.UpdateStages.WithIgnoreMissingVnetServiceEndpoint"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRule.UpdateStages.WithVirtualNetworkSubnetId"
- "com.azure.resourcemanager.postgresql.models.VirtualNetworkRules"
metadata: {}
package: "com.azure.resourcemanager.postgresql.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-postgresql:1.0.0-beta.1
