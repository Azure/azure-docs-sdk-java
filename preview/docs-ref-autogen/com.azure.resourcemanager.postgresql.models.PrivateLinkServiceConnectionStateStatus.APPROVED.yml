### YamlMime:JavaMember
uid: "com.azure.resourcemanager.postgresql.models.PrivateLinkServiceConnectionStateStatus.APPROVED*"
fullName: "com.azure.resourcemanager.postgresql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStateStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.postgresql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
  fullName: "com.azure.resourcemanager.postgresql.models.PrivateLinkServiceConnectionStateStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStateStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStateStatus."
  syntax: "public static final PrivateLinkServiceConnectionStateStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.postgresql.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-postgresql:1.0.0-beta.1
